src/Bindings/Browser/VersioningService.php

```diff
@@ -56,7 +66,7 @@ public function cancelCheckOut($repositoryId, $objectId, ExtensionDataInterface
      */
     public function checkIn(
         $repositoryId,
-        $objectId,
+        & $objectId,
```

Comment:
Style is copied from https://github.com/dkd/php-cmis-client/blob/master/src/Bindings/Browser/VersioningService.php#L84 (should we change that across the repository in another commit?)